pages/dkp/kommander/2.2/clusters/attach-cluster/manually-attach-cluster/index.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: layout.pug
+navigationTitle: Manually attach Cluster created with CLI
+title: Attach a Cluster manually created with CLI
+menuWeight: 50
+excerpt: Attach a Cluster manually created with CLI
+---
+
+When creating a cluster in a Workspace namespace using the DKP CLI it will not be attached automatically. To automatically attach a cluster you can generate the cluster objects using the DKP CLI `--dry-run -o yaml` flags and then using the [Advanced Creation of Konvoy Clusters][create_cluster_advanced].
```

Comment:
```suggestion
When you create a cluster in a Workspace namespace using the DKP CLI, it does not attach automatically. To automatically attach a cluster, generate the cluster objects using the DKP CLI `--dry-run -o yaml` flags and create a cluster following the instructions in the [Advanced Creation of Konvoy Clusters][create_cluster_advanced] guide.
```

Reply:
Would this solve the problem for good? If a user does this, he/she won't have to manually attach clusters created afterwards? If yes, we can take my suggestion to make that a bit more clear. 
If this is a step the user needs to follow every time, we could consider including this info in the steps.

Reply:
This is the alternative, if you generate the yaml using the CLI with `--dry-run` then use the UI to create the cluster, it **will** automatically attach.

These steps are only needed if you use the CLI to create the cluster. And this won't be required in 2.3.

Reply:
Awesome, thank you @TattdCodeMonkey . In that case, I think we could use the following. Does it sound good?
```suggestion
When you create a cluster in a Workspace namespace using the DKP CLI, it does not attach automatically. To automatically attach a cluster, generate the cluster objects using the DKP CLI `--dry-run -o yaml` flags and create a cluster as stated in the [Advanced Creation of Konvoy Clusters][create_cluster_advanced] guide.
```